index.css

```diff
@@ -60,4 +58,9 @@ body {
 .roborox-logo {
     width: 50px;
     height: auto;
-}
\ No newline at end of file
+}
+
+ 
+/* This is the way I found that might work for making the top search form to the right of the roborox text, but is there a better way? I don't know if this will work on every resolution
```

Comment:
Yes. This doesn't seem to work on every resolution.  When I reduce the size of the browser window to around 990px width (the point where the burger icon shows up),  the search textbox is forced over to the right.  You can try to do a float-left;  in a css definition like so:  

```
#header-search {
    float: left;
}
```

Reply:
actually been trying a couple of things.  I did remove d-flex.  

ex. of the issue:
![image](https://user-images.githubusercontent.com/651674/125303747-05fbf880-e2fb-11eb-856d-b577aaa5c833.png)